rest_framework/templates/rest_framework/base.html

```diff
@@ -2,33 +2,43 @@
 {% load rest_framework %}
 {% load i18n %}
 
+
+{% block prehead %}
```

Comment:
Why would you not allow template to be overridden for those who would need that? By default it would stay as it is currently. But it opens new opportunities and possibilities.